### PR TITLE
remove duration from lock delay as seconds are the only granularity s…

### LINKED ISSUE #2153: consul_session: consul.session.create() requires a lock_delay without units
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

consul_session
##### ANSIBLE VERSION

```
ansible 2.0.2.0
  config file =
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

N/A
##### OS / ENVIRONMENT

```
pip freeze | grep consul
python-consul==0.6.0
```
##### SUMMARY

When creating a consul session, the module requires passing a unit with the `lock_delay` parameter. This is not a valid option in the [underlying python-consul module](http://python-consul.readthedocs.io/en/latest/#consul.base.Consul.Session), which states `lock_delay is an integer of seconds`.

Fix here: https://github.com/quantopian/ansible-modules-extras/commit/763c8f75c19b926618d1a066ec792c058872fd43  
##### STEPS TO REPRODUCE

``` yaml
    - name: Register new session
      consul_session:
        name: "{{ consul_session_name }}"
```

`ansible-playbook -i 'localhost,' -c local consul.yml`
##### EXPECTED RESULTS
- successful playbook execution
- successful creation of a named session in consul
##### ACTUAL RESULTS

```
fatal: [ec2.compute-1.amazonaws.com]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_args": {"checks": null, "datacenter": null, "delay": "15s", "host": "localhost", "id": null, "name": "foo", "node": null, "port": 8500, "state": "present"}, "module_name": "consul_session"}, "msg": "Could not create/update session No JSON object could be decoded"}
```

A local debug session provided the error:

```
[10] > /usr/lib/python2.7/json/decoder.py(384)raw_decode()
-> raise ValueError("No JSON object could be decoded")
(Pdb++) locals()
{'s': u'Request decode failed: time: unknown unit ss in duration 15ss', 'self': <json.decoder.JSONDecoder object at 0x7f9a3bb48ed0>, 'idx': 0}
```

### PR DESCRIPTION
**Pull request by sgargan**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

consul_session
##### ANSIBLE VERSION

```
ansible 2.2.0 (consul-test-updates fb09c88f40) last updated 2016/09/07 21:50:29 (GMT +100)
  lib/ansible/modules/core:  not found - use git submodule update --init lib/ansible/modules/core
  lib/ansible/modules/extras:  not found - use git submodule update --init lib/ansible/modules/extras
  config file = /Users/sgargan/.ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Looking into this again @irlevesque was correct about this and the units expected. The service expects a lock delay in seconds.

fixes #2153

add utf header to file so that it loads correctly

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @sgargan. This module is going into community review.

We notice that you are one of the maintainers (@sgargan) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by sgargan** [comment]
:shipit:

File: clustering/consul_session.py [hunk 1]
```diff
@@ -259,7 +249,7 @@ def test_dependencies(module):
 def main():
     argument_spec = dict(
         checks=dict(default=None, required=False, type='list'),
-        delay=dict(required=False,type='str', default='15s'),
+        delay=dict(required=False,type='str', default='15'),
```
Comment (resmo) [inline clustering/consul_session.py]: wondering if it would make sense to use int?

**Comment by gregdek** [comment]
Thanks @sgargan. Since you are one of the maintainers (@sgargan) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: clustering/consul_session.py (hunk 1 above)
Comment (sgargan) [reply clustering/consul_session.py]: it absolutely would, I've made the change.

**Pull request merged**